launch.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+export QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
```

Comment:
Why aren't you adding instead `--env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess` to the `finish-args` in the manifest?

Reply:
No particular reason. I just didn't know about it (sorry about that).
If you have any other observations, please do.
This is my first time messing with flatpak, so there may be lots of things to improve.